### ISSUE TITLE
Fragments fail to attach after replacrFragment

### ISSUE THREAD

**Issue by nikhil-thakkar** [ncapdevi/FragNav#194]
This is a strange issue I'm facing.
Steps to reproduce:
1. Clearstack for current tab and replace the root fragment using replaceFragment method.
2. Though the above step works but when you switch tabs, the already attached fragment disappears and fails to attach itself.

Please look into this issue and let me know if this sounds valid.

**Comment by nikhil-thakkar** [comment]
To add to, my fragmentHideStrategy was DETACH_ON_NAVIGATE_HIDE_ON_SWITCH but when I change it to DETACH, the fragments are attach/detach correctly.

**Comment by ncapdevi** [comment]
Hmm, just to clarify, are the `clearstack` and `replaceFragment` methods called sequentially and immediately after each other?  Also could you verify which version of FragNav you're using.

**Comment by nikhil-thakkar** [comment]
Yes, the clearstack and replaceFragment are being called sequentially and immediately.
The version of FragNav we are using is 3.1.0.

**Comment by Onum** [comment]
Same issue. When can we expect a fix?

**Comment by AEslamifar** [comment]
Same issue.

**Comment by bvk070** [comment]
Same issue. It's been more than a year. 
When can we expect a fix?
